src/test/java/com/wojteknier03/clinicmedical/appointment_service/AppointmentServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package com.wojteknier03.clinicmedical.appointment_service;
+
+
+import com.wojteknier03.clinicmedical.dto.AppointmentDto;
+import com.wojteknier03.clinicmedical.mapper.AppointmentMapper;
+import com.wojteknier03.clinicmedical.model.Appointment;
+import com.wojteknier03.clinicmedical.model.Patient;
+import com.wojteknier03.clinicmedical.repository.AppointmentRepository;
+import com.wojteknier03.clinicmedical.repository.PatientRepository;
+import com.wojteknier03.clinicmedical.service.AppointmentService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import org.mockito.Mockito;
+import org.springframework.data.domain.Pageable;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.Mockito.when;
```

Comment:
zauwaz ze package w ktorym sie znajduje AppointmentService to package service a nie package appointment_service dodatkowo zauwaz ze masz cos zepsute przeniesc package model do packageu clinicmedical w ktorym jest reszta rzeczy